bazel/repository_locations.bzl

```diff
@@ -846,6 +846,7 @@ REPOSITORY_LOCATIONS_SPEC = dict(
             "envoy.filters.http.rbac",
             "envoy.filters.http.wasm",
             "envoy.filters.network.rbac",
+            "envoy.rbac.matchers.upstream",
```

Comment:
I don't think this change and below should be needed. You shouldn't be using cel or flatbuffers from these extensions. What is the error you get if you don't have this?

Reply:
I tried removing them but then get errors in deps CI build. I think the dependencies are brought in by the matcher library which depends on common expression library.